libr/core/project.c

```diff
@@ -478,17 +472,12 @@ static bool store_files_and_maps(RCore *core, RIODesc *desc, ut32 id) {
 	if (desc) {
 		// reload bin info
 		r_cons_printf ("\"obf %s\"\n", desc->uri);
-		r_cons_printf ("\"of \\\"%s\\\" %s\"\n", desc->uri,
-				r_str_rwx_i (desc->perm));
+		r_cons_printf ("\"of \\\"%s\\\" %s\"\n", desc->uri, r_str_rwx_i (desc->perm));
 		if ((maps = r_io_map_get_by_fd (core->io, id))) { //wtf
 			r_list_foreach (maps, iter, map) {
-				r_cons_printf ("om %d 0x%" PFMT64x " 0x%"
-						PFMT64x " 0x%" PFMT64x
-						" %s%s%s\n", fdc,
-					r_io_map_begin (map), r_io_map_size (map),
-					map->delta, r_str_rwx_i (map->perm),
-					map->name ? " " : "",
-					r_str_get (map->name));
+				r_cons_printf ("om %d 0x%" PFMT64x " 0x%" PFMT64x " 0x%" PFMT64x " %s%s%s\n", fdc, r_io_map_begin (map),
```

Comment:
Even if that fits in 2 lines i think its less readable . Thats why automatic indentation will never be better than a human because it cant appreciate or determine when a construction is easier for the eye or not. Same as following rules blindly